src/wp-includes/block-patterns.php

```diff
@@ -226,14 +250,14 @@ function _load_remote_featured_patterns() {
 		return;
 	}
 	$patterns = $response->get_data();
-
+	$registry = WP_Block_Patterns_Registry::get_instance();
 	foreach ( $patterns as $pattern ) {
-		$pattern_name = sanitize_title( $pattern['title'] );
-		$registry     = WP_Block_Patterns_Registry::get_instance();
+		$normalized_pattern = _normalize_remote_pattern( $pattern );
+		$pattern_name       = sanitize_title( $normalized_pattern['title'] );
 		// Some patterns might be already registered as core patterns with the `core` prefix.
 		$is_registered = $registry->is_registered( $pattern_name ) || $registry->is_registered( "core/$pattern_name" );
 		if ( ! $is_registered ) {
-			register_block_pattern( $pattern_name, (array) $pattern );
+			register_block_pattern( $pattern_name, (array) $normalized_pattern );
```

Comment:
```suggestion
			register_block_pattern( $pattern_name, $normalized_pattern );
```
The type cast is not needed because `_normalize_remote_pattern()` will always return an array data type.